Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -150,29 +150,29 @@
   components:
   - type: StorageFill
     contents:
-      - id: ClothingOuterHardsuitEngineeringWhite
+#      - id: ClothingOuterHardsuitEngineeringWhite
```

Comment:
dont do this please. Its literally a 'hardsuit filled' version and you're removing the hard suit.

Reply:
Used wrong belt by mistake.